examples/md-flexible/input/AllOptions.yaml

```
@@ -11,8 +11,10 @@ tuning-strategies                    :  [slow-config-filter, rule-based-tuning,
 tuning-interval                      :  5000
 tuning-samples                       :  3
 tuning-max-evidence                  :  10
+tuning-metric                        :  energy
```

```suggestion
tuning-metric                        :  time # alternatively: energy
```
I know `AllOptions.yaml` is not really supposed to be "default" options (whatever that means), but I'd prefer this still.